stock_orderpoint_uom/tests/test_stock_orderpoint_procure_uom.py

```diff
@@ -25,24 +31,31 @@ def setUp(self):
              'type': 'product',
              'uom_id': self.uom_unit.id,
              'default_code': 'A',
+             'variant_seller_ids': [(6, 0, [self.vendor.id])],
              })
 
     def test_stock_orderpoint_procure_uom(self):
 
-        self.env['stock.warehouse.orderpoint'].create({
+        orderpoint = self.env['stock.warehouse.orderpoint'].create({
             'warehouse_id': self.warehouse.id,
             'location_id': self.location_stock.id,
             'product_id': self.productA.id,
             'product_max_qty': 24,
             'product_min_qty': 12,
+            'product_uom': self.uom_unit.id,
             'procure_uom_id': self.uom_dozen.id,
         })
 
-        sched = self.env['procurement.order']
-        sched.run_scheduler()
-        proc = sched.search([('product_id', '=', self.productA.id)])
-        self.assertEqual(proc.product_uom, self.uom_dozen)
-        self.assertEqual(proc.product_qty, 2)
+        self.env['procurement.group'].run_scheduler()
+        # As per route configuration, it will create Purchase order
+        purchase = self.purchase_model.search(
+            [('origin', 'ilike', orderpoint.name)])
+        self.assertEquals(len(purchase), 1)
+        purchase_line = self.purchase_line_model.search(
+            [('orderpoint_id', '=', orderpoint.id),
+             ('order_id', '=', purchase.id)])
+        self.assertEquals(len(purchase_line), 1)
+        self.assertEqual(purchase_line.product_qty, 2.0)
```

Comment:
use `assertAlmostEqual` for float comparsions

Reply:
I have changed from _2.0_ to _2_